File: app/src/main/java/com/dhsdevelopments/potato/clientapi/gcm/GcmRegistrationRequest.java
package com.dhsdevelopments.potato.clientapi.gcm;

import com.google.gson.annotations.SerializedName;

public class GcmRegistrationRequest
{
    @SerializedName( "token" )
    public String token;

    public GcmRegistrationRequest() {
    }

    public GcmRegistrationRequest( String token ) {
        this.token = token;
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/gcm/GcmRegistrationRequest.kt
package com.dhsdevelopments.potato.clientapi.gcm

import com.google.gson.annotations.SerializedName

class GcmRegistrationRequest {
    @SerializedName("token")
    lateinit var token: String

    constructor() {
    }

    constructor(token: String) {
        this.token = token
    }
}
